packages/core/src/renderTexture/RenderTexturePool.ts

```diff
@@ -78,8 +78,8 @@ export class RenderTexturePool
     {
         let key;
 
-        minWidth = Math.ceil(minWidth * resolution);
-        minHeight = Math.ceil(minHeight * resolution);
+        minWidth = Math.ceil((minWidth * resolution) - 1e-6);
```

Comment:
Don't we have an EPSILON setting / constant?

Reply:
We do not have a consistent EPSILON. I found two usages: `Number.EPSILON` is being used in math-extras, and `GRAPHICS_CURVES.epsilon` in the graphics. None of them are consistent. I think this is okay here.

Reply:
We have https://github.com/pixijs/pixijs/blob/d1e9cf2a069f07d355b09b84afbb250bb8cc8817/packages/math/src/shapes/Rectangle.ts#L198